LoveMachine.Core/ButtplugController.cs

```diff
@@ -37,6 +37,7 @@ public void OnEndH()
                 foreach (var bone in GetSupportedBones(girlIndex))
                 {
                     DoVibrate(0f, girlIndex, bone);
+                    HandleCoroutine(DoRotate(girlIndex, bone, true, 0f));

```

Comment:
@Sauceke Sorry, I'm not familiar with C# though, is this implementation not enough to stop rotators?

https://github.com/Sauceke/LoveMachine/pull/97#discussion_r891205169
https://github.com/Sauceke/LoveMachine/pull/97#discussion_r895216141

I had added this logic but it still doesn't stop sometimes.

Reply:
This will start a coroutine in the same frame StopAllCoroutines() is called. I'm not sure how Unity handles that.
Have you tried calling the client instead?
```suggestion
                    client.RotateCmd(0f, clockwise, girlIndex, bone);
```

Reply:
@Sauceke Thanks, the code works well.